internal.nix

```diff
@@ -241,7 +241,8 @@ rec {
   sourceHashFunc = githubSourceHashMap: spec:
     if spec.type == "github" then
       lib.attrByPath [ spec.value.org spec.value.repo spec.value.rev ] null githubSourceHashMap
-    else null;
+    else
+      throw "sourceHashFunc: spec.type '${spec.type}' is not supported. Supported types: 'github'";
```

Comment:
Missing the `[npmlock]` prefix. Perhaps (at some point) we want to declare a custom `throw` function that does it for us.

Reply:
ah, good catch..

Reply:
i created https://github.com/tweag/npmlock2nix/issues/76 for that so we won't forget about that